test/CMakeLists.txt

```diff
@@ -337,7 +337,7 @@ endforeach(FILENAME)
 # iodaconv_comp.sh test.
 #
 # For a converter that is generating new data, use a non-zero tolerance.
-set(IODA_CONV_COMP_TOL_ZERO "0.0")
+set(IODA_CONV_COMP_TOL_ZERO "1.e-7")
```

Comment:
@PatNichols do not think this is the way to go... this definition should remain zero

my approach was to change those tests to use the other tolerance:

`IODA_CONV_COMP_TOL` please see my PR #1064